src/elements/table/table-wrapper/table-wrapper.component.ts

```diff
@@ -22,21 +24,68 @@ export
 class SbbTableWrapperElement extends SbbNegativeMixin(LitElement) {
   public static override styles: CSSResultGroup = style;
 
-  private _resizeObserver = new ResizeController(this, {
-    target: null,
-    skipInitial: true,
-    callback: () => this._checkHorizontalScrollbarOffset(),
-  });
-  private _tableWrapper!: HTMLElement;
+  /** Whether the table wrapper is focusable. */
+  @forceType()
+  @property({ reflect: true, type: Boolean })
+  public accessor focusable: boolean = false;
```

Comment:
It might be that I'm not understanding the scope completely, but at the moment the table is always focusable, whether `focusable` is set to `true` or not. Should it naturally prevent focus (maybe with a tabindex="-1") when it's set to false?

Reply:
This is due to every scroll area (overflow: auto) is per definition focusable as long as there is no focusable element within. But I think this behavior should not be overridden by us. It could break accessibility. So if you expand your window, and there is no scroll area anymore, it shouldn't be focusable (when property turned off)?

Reply:
Makes sense, thanks